src/norm.jl

```diff
@@ -0,0 +1,92 @@
+# Code adapted from Flux.jl:
+# https://github.com/FluxML/Flux.jl/blob/68ba6e4e2fa4b86e2fef8dc6d0a5d795428a6fac/src/layers/normalise.jl#L117-L206
+# License: https://github.com/FluxML/Flux.jl/blob/master/LICENSE.md
+
+mutable struct BatchNormFlow{T1,T2} <: Bijector
+    β::T1
+    logγ::T1
+    μ  # moving mean
+    σ² # moving st
+    ϵ::T2
+    momentum::T2
+    active::Bool # true when training
+end
+
+BatchNormFlow(dims::Int, container=Array; ϵ=1f-5, momentum=0.1f0) = BatchNormFlow(
```

Comment:
Is the `container=Array` purposefully not part of the keyword arguments here or should it be after `;`?